src/aiai_eval/task_configs.py

```diff
@@ -234,6 +234,32 @@ def get_all_task_configs() -> Dict[str, TaskConfig]:
     test_name="test",
 )
 
+DISCOURSE = TaskConfig(
+    name="discourse-coherence-classification",
+    huggingface_id="ajders/ddisco",
```

Comment:
Should be changed when the dataset is moved.
```suggestion
    huggingface_id="alexandrainst/ddisco",
```